### PR TITLE
File Integrity: add blake3 support

### LINKED ISSUE #41: Help offered: File Integrity Plugin update from blake2 to blake3
Hi, does anybody like to have this feature?

https://github.com/BLAKE3-team/BLAKE3/releases/tag/1.0.0

Performance improvements are huge.

Do you like to get a pull request from me for this update? Or is already somebody working on it?
If somebody did evaluate and postponed this task, I would like to know the reason and reevaluate it.

Based on your feedback, I will share my work or keep it for myself.

### PR DESCRIPTION
**Pull request by Falcosc**
Resolves #41

I don't know how to create the dynamix.file.integrity.txz, which means the MD5 sum for the installation needs to added after my pullrequest.

But I guess you want to create dynamix.file.integrity.txz by yourself anyway to make sure it is equal to the git content.

### CODE REVIEW

File: unRAIDv6/dynamix.file.integrity.plg [hunk 1]
```diff
@@ -319,6 +322,8 @@ if [[ -z $method ]]; then
   hash=sha256
 elif [[ $method == -b2 ]]; then
   hash=blake2
+elif [[ $method == -b3 ]]; then
```
Comment (Falcosc) [inline unRAIDv6/dynamix.file.integrity.plg:325]: I don't know where the $method is coming from, so I am not sure if I broke this piece of code.

File: unRAIDv6/dynamix.file.integrity.plg [hunk 2]
```diff
@@ -3,11 +3,11 @@
 <!DOCTYPE PLUGIN [
 <!ENTITY name      "dynamix.file.integrity">
 <!ENTITY author    "Bergware">
-<!ENTITY version   "2020.06.21">
+<!ENTITY version   "2021.08.01">
 <!ENTITY launch    "Settings/FileIntegrity">
 <!ENTITY pluginURL "https://raw.githubusercontent.com/bergware/dynamix/master/unRAIDv6/&name;.plg">
 <!ENTITY source    "/boot/config/plugins/&name;/&name;">
-<!ENTITY MD5       "0c2a39f5724b3c0529aec14469b1ea8b">
+<!ENTITY MD5       "TODO">
```
Comment (Falcosc) [inline unRAIDv6/dynamix.file.integrity.plg:10]: Needs to be done by you

**Comment by Falcosc** [comment]
No negative effects if you are IO limited:

BLAKE3: 178809 files. Duration: 02:16:36. Average speed: 59.1 MB/s
BLAKE2: 178809 files. Duration: 02:20:39. Average speed: 57.4 MB/s

I was worried a bit if it is ok to call b3sum 180.000 times instead of calling it only once. 
Conclusion: The new b3sum CLI is as lightweight as the old b2sum.

**Comment by Falcosc** [comment]
This Pull request does mainly reduce power consumption by reducing CPU time.
@bergware is the description ok, or is it misleading because users of spinning disks are IO limited and will not expect any speed improvements?

**Comment by bergware** [comment]
Thanks for your updates, these coming days I'll do some testing and merge them for a new release.

File: unRAIDv6/dynamix.file.integrity.plg (hunk 1 above)
Comment (Falcosc) [reply unRAIDv6/dynamix.file.integrity.plg:325]: Found and test it.

**Comment by Falcosc** [comment]
How should we go on with the package definition?

It is compatible with the old Version, so I could just revert the MD5 checksum and Version String change and than it would be safe to merge.

**Comment by Falcosc** [comment]
@bergware I figured out how you do release things.
My solution was to add my change log above the current release without defining a new release.
My section is now called `comming soon`

**Comment by bergware** [comment]
My plan is coming weekend to merge your updates and soon after that release a new version.

Thx for your contributions.

**Comment by Falcosc** [comment]
@bergware Some pull requests will have merge issues, even if you merge in correct order. I will rebase all broken ones after your merges.

**Pull request closed without merging**